### PR TITLE
feat(config): add evmos production to config

### PR DESCRIPTION
**Pull request by ltchang2019**

### CODE REVIEW

**Review by yourbuddyconner — APPROVED** [review]
LGTM

File: configuration/configs/production.json [hunk 1]
```diff
@@ -538,6 +719,12 @@
       "connections": ["moonbeam", "milkomedaC1"],
```
Comment (Imti) [inline configuration/configs/production.json:719]: Need to add `Evmos` here as well

File: configuration/configs/production.json [hunk 2]
```diff
@@ -538,6 +714,12 @@
...
       "manualProcessing": true,
       "connextEnabled": true
+    },
+    "evmos": {
+      "displayName": "Evmos",
+      "nativeTokenSymbol": "EVMOS",
+      "connections": ["ethereum"],
+      "connextEnabled": true
```
Comment (ErinHales) [inline configuration/configs/production.json:722]: I think this should be false initially.  Don't know what the state of Connext deploys are on evmos

**Review by Imti — APPROVED** [review]
🔑

**Pull request merged**